src/components/Accordion/Accordion.stories.js

```diff
@@ -25,6 +25,10 @@ const props = {
   style: { width: '100vw' },
 };
 
+const story = 'accordion';
+const description =
+  'Accordions allow users to expand and collapse sections of content.';
+
```

Comment:
Could you add a display name for the `Select` used in this component demo, so that it doesn't show as `[object Object] in the story source?

```suggestion

Select.displayName = 'Select';
```

Reply:
🤔 I guess this is still a symptom of the maintenance issue we're having! I'll spend a bit more time debugging this, but otherwise, if it's the same build issue, we're going to need to do this for every component we proxy

Reply:
I guess this is still a symptom of the maintenance issue we're having! I'll spend a bit more time debugging this, but otherwise, if it's the same build issue, we're going to need to do this for every component we proxy

Reply:
Resolving this convo since the fix was proposed upstream in Carbon 👍